openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/raw-wb-board.js

```diff
@@ -369,6 +370,52 @@ var Wb = function() {
 					minHeight: 140
 					, minWidth: 255
 				});
+				tools.find('.om-icon.dweet').click(function(){					
+			        dweet = OmUtil.tmpl("#wb-dweet");			        
+			       	dweet[0].id='wb-dweet' + dweetIDCount++;        
```

Comment:
It is no clear why element ID should be different every time user role is changed

Reply:
Everytime the user clicks on the dweet button a new dweet popup will be opened. One user can have multiple dweet popup's opened to control different IOT things. So giving each popup a unique id so that on submit can handle the specific popup action.